docs/section12/Rule12-1.md

```diff
@@ -15,24 +15,20 @@
 
 ## Rule Logic:  
 
-- Get the total number of spaces in the building segment in the User model: ```For building_segment in U_RMR.building.building_segments:```  
+- Get the total number of spaces in the building segment in the User model: ```for building_segment_user in U_RMR.building.building_segments:```  
 
```

Comment:
@xingcx can we start with 'for each building in U-RMR. There might be a scenario of multiple buildings in a RMR and the rulee definition should loop through each building as well.